internal/components/sidecarinjector/testdata/icp-expected-values.yaml

```diff
@@ -1,6 +1,6 @@
-revision: cp-v113x
+revision: cp-v115x
 deployment:
-  image: banzaicloud/istio-sidecar-injector:v1.10.4-bzc.1
+  image: banzaicloud/istio-sidecar-injector:v1.15.0-bzc.1
```

Comment:
Does this image actually exist?

Reply:
No, not yet

Reply:
And will it? Seems like we haven't released a new one since `1.10.4`, will we have a new one for some reason?

Reply:
This is only testdata, no real images will be downloaded.

Reply:
oh, okay :)